src/tutorials/simple-service-provider.md

```diff
@@ -724,163 +718,70 @@ Add the following block of functions and variable into the `index.ts`:
+Lets get get our __Service Provider's__ Websocket Client running.
 
-<img src="../images/tutorial_image_2.png"/>
+1.  To get the __Service Provider's__ Websocket Client running, navigate to `path/to/the/release` folder (like we did with our __User Client__ instance) and run the following to initialize 
+    your second `nym-client`: 
 
-The Service Provider is seeking a websocket connection on port `1978` but currently it cannot find it. 
+    ```
+    ./nym-client init --id service-provider --port 1978
+    ```
 
-2. Open a new terminal window, and `path/to/the/release` folder, and run the following to initialize your first `nym-client`: 
+    We want to specify port `1978` since thats the port specified in our `index.ts`. Different `nym-client`'s must run on different ports in order to avoid conflicts.
 
-```
-./nym-client init --id service-provider --port 1978
-```
-
-3. Then run:
-```
-./nym-client run --id service-provider
-```
+3.  Then run:
 
-The second Nym Websocket Client is now running, so if you return to the Service Provider tab in your browser and refresh, you'll see the following:
-
-<img src="../images/tutorial_image_3.png"/>
+    ```
+    ./nym-client run --id service-provider
+    ```
 
-Similar to the User Client, in the 'Activity Log' of the UI, there's a successful response from our websocket, thus we're able to see the same address from our terminal. If we were to terminate our `nym-client`, we can an error on the browser UI stating a missing websocket connection. This is a good sign of error handling.
+    The second Nym Websocket Client is now running!
 
-4. The final step of this tutorial is to update the `targetAddress` in the User Client's `index.ts`. Assign the global variable `targetAddress` we initialized, with the address of the Service Provider's Nym Websocket client.
+    <img src="../images/tutorial_image_4.png"/>
 
-```
-var targetAddress = '<service-provider-websocket-client-addresss>';
-```
+    We can see that we get our Nym Websocket Address successfully.
 
-> ⚠️ Reminder that the client address produced from running this command in your terminal will always be different to the one you have when you execute this command. Each address generated by each client will be different.
+4.  The final step of this tutorial is to update the `targetAddress` in the __User Client__'s `index.ts`. Assign the global variable `targetAddress` we initialized, with the address of the
+    __Service Provider's__ Nym Websocket client. Feel free to copy and paste the resulting address that you see in your terminal (purple text).
 
-> ⚠️ Make sure to save all files and refresh browsers before proceeding! 
+    ```
+    var targetAddress = '<service-provider-websocket-client-addresss>';
+    ```
 
+    > ⚠️ Reminder that the client address produced from running this command in your terminal will always be different to the one you have when you execute this command. Each address generated by each client will be different.
```

Comment:
Why will it be different? What makes up a client address? Link to the docs + do grammar fixes